vuepress/docs/_posts/2023-08-09-install-entando-with-docker-desktop.md

```diff
@@ -0,0 +1,107 @@
+
+# Install the NGINX Ingress Controller
+Entando uses a small set of paths for the different services in an Entando Application so we'll need to install an ingress controller. We'll use the NGINX version since it is used in most of the [Entando install guides](../v7.2/tutorials/README.md)
+
+1. Enable the NGINX ingress controller [following the steps here](https://kubernetes.github.io/ingress-nginx/deploy/#docker-desktop):
+``` bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.8.1/deploy/static/provider/cloud/deploy.yaml
+```
+
+2. Access an arbitrary URL against localhost and you should now see an NGINX 404 error, for example `http://localhost/some-path`.
+
+# Install Entando
+Many of the following steps are identical to those in the [Getting Started guide](../v7.2/docs/getting-started/README.md) so see that page for explanatory details. 
+> *Note:* Make sure to check for the correct Entando patch version for the install steps since this blog post specifically documents the steps for Entando 7.2.2.
+
+1. Create the namespace:
+``` bash
+kubectl create namespace entando
+```
+
+2. Download the operator config map template:
+``` bash
+curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.2.2/dist/ge-1-1-6/samples/entando-operator-config.yaml"
+```
+
+3. Edit the file to select the NGINX ingress controller:
+``` yaml
+data: 
+  entando.ingress.class: "nginx"
+```
+
+4. Create the operator `ConfigMap`:
+``` bash
+kubectl apply -f entando-operator-config.yaml -n entando
+```
+
+5. Add the cluster resources:
+``` bash
+kubectl apply -f https://raw.githubusercontent.com/entando/entando-releases/v7.2.2/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
+```
+
+6. Add the namespace resources:
+``` bash
+kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v7.2.2/dist/ge-1-1-6/namespace-scoped-deployment/namespace-resources.yaml
+```
+
+7. Determine your IP address (YOUR-IP) using `hostname -I | awk '{print $1}'` or by visiting `https://whatismyip.com`.
+
+8. Download the Entando Application template
+``` bash
+curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.2.2/dist/ge-1-1-6/samples/entando-app.yaml"
+```
+
+9. Edit the template and set the ingressHostName to `entando.YOUR-IP.nip.io` or another address that will route to your local machine.
+> *Note:* Some connections from one pod to another will use a public address, rather than a cluster-level address. If the component manager and de-app deployments fail to start correctly, this is the most likely cause. See the [tips and tricks page](../v7.2/docs/reference/local-tips-and-tricks.md) for troubleshooting tips. In my testing I realized my router had recently upgraded and was blocking all port forwarding so I had to re-enable that setting.
```

Comment:
```suggestion
> *Note:* Some connections from one pod to another will use a public address, rather than a cluster-level address. If the Component Manager and `entando-de-app` deployments fail to start correctly, this is the most likely cause. See the [tips and tricks page](../v7.2/docs/reference/local-tips-and-tricks.md) for troubleshooting tips. In my testing, I realized my router had recently upgraded, blocking all port forwarding, so I had to re-enable that setting.
```